addons/azureSets/azuresets.lua

```diff
@@ -191,17 +212,13 @@ function set_spells_from_spellset(spellset, setPhase)
 end
 
 function find_spell_id_by_name(spellname)
-    for spell in spells:it() do
-        if spell['english']:lower() == spellname:lower() then
-            return spell['id']
-        end
+    local spell = spellsLookup[spellname] or spellsLookup[spellname:lower()]
```

Comment:
If you always use `:lower()` as I did above you can drop the first check. The entire function could be changed to this:
```lua
function find_spell_id_by_name(name)
    local spell = spellsLookup[name:lower()]
    return spell and spell.id
end
```